config/routes.rb

```diff
@@ -1,5 +1,14 @@
 FloodRiskEngine::Engine.routes.draw do
   resources :enrollments, only: [:new, :create] do
     resources :steps, only: [:show, :update], controller: "enrollments/steps"
+
+    member do
```

Comment:
State machine isn't involved - this is the steps controller. 

My feeling was that for one action, it wasn't worth the extra work of creating another controller. However, I'm happy to be persuaded otherwise (which seems to be the case from the discussion below)